mne/utils/check.py

```diff
@@ -958,6 +958,44 @@ def _check_sphere(sphere, info=None, sphere_units='m'):
     return sphere
 
 
+def _check_head_radius(radius, add_info=''):
+    """Check that head radius is within a reasonable range (6.65 - 10.85 cm).
+
+    Parameters
+    ----------
+    radius : float
+        Head radius in meters.
+    add_info : str
+        Additional info to add to the warning message.
+
+    Notes
+    -----
+    The minimum and maximum values were taken from the head size percentiles
+    given in the following Wikipedia infographic:
+    https://upload.wikimedia.org/wikipedia/commons/0/06/AvgHeadSizes.png
+    the minimum radius is taken from the 1st percentile for women bitragion
+    breadth (head breadth not including the ears):
+
+        12.5 cm / 2 = 6.25 cm = 0.0625 m
+
+    while the maximum radius is taken from the 99th percentile for men Glabella
+    to back of the head measurements (Glabella is a point just above the
+    Nasion):
+
+        21.7cm / 2 = 10.85 cm = 0.1085 m
+    """
+    min_radius = 0.0625
```

Comment:
FYI I work with a lot of infant data and I was worried about getting warnings with them, but I did a dozen quick fits to 6 month olds and all were at least 0.065, so I suppose this is small enough even for them? But feel free to adjust lower if you hadn't thought about this use case and want to be more conservative

Reply:
I considered this case, but the warning used to say "for adult head size" and I retained that. But I found this [source for infant head diameter](https://www.cdc.gov/growthcharts/html_charts/hcageinf.htm) and I can change the min radius to a value corresponding to 3rd percentile of female 0-month infant head circumference (that would be 5.08 cm, so I can round it down to 5 cm).

Reply:
Sure let's round down to 0.05. Order of magnitude errors with adults should still show up even with this more lenient limit and we'll get fewer (no?) false alarms with infants that way